book/src/testnet-validator.md

```diff
@@ -0,0 +1,162 @@
+# Become a Testnet Validator
+
+Joining an Eth2 testnet is a great way to get familiar with staking in Phase 0.
+All users should experiment with a testnet prior to staking mainnet ETH.
+
+## Supported Testnets
+
+Lighthouse supports four testnets:
+
+- [Medalla](https://github.com/goerli/medalla/tree/master/medalla) (default)
+- [Zinken](https://github.com/goerli/medalla/tree/master/zinken)
+- [Spadina](https://github.com/goerli/medalla/tree/master/spadina) (deprecated)
+- [Altona](https://github.com/goerli/medalla/tree/master/spadina) (deprecated)
+
+When using Lighthouse, the `--testnet` flag selects a testnet. E.g.,
+
+- `lighthouse` (no flag): Medalla.
+- `lighthouse --testnet medalla`: Medalla.
```

Comment:
We could replace `--testnet` with `--network` in v0.3.0 since it's a breaking change. See https://github.com/sigp/lighthouse/issues/1689

Reply:
I *might* be able to get this done before v0.3.0, we'll see.